src/analyticsTracking/AnalyticsTrackingProvider.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {AMPLITUDE_KEY} from '../common/globals';
+import {AmplitudeProvider} from '@amplitude/react-amplitude';
+import amplitude from 'amplitude-js';
+import AnalyticsTrackingContext from './AnalyticsTrackingContext';
+import AmplitudeProxy from './AmplitudeProxy';
+import analyticsTrackingConstants from '../common/analyticsTrackingConstants';
+import AmplitudeUser from './AmplitudeUser';
+
+const {AUTH} = analyticsTrackingConstants;
+
+let amplitudeUser = null;
+// log amplitude users on auto-login
+if (localStorage.getItem('idToken')) {
+  amplitudeUser = new AmplitudeUser(
+    localStorage.getItem('idToken'),
+    AMPLITUDE_KEY,
+  );
+
+  amplitudeUser.instance.logEvent(AUTH.LOGIN, {
```

Comment:
correct, at least every page refresh

Reply:
@dankolbman can you think of a way to only fire it when necessary?

Reply:
I don't have to many ideas outside of storing something else locally. Although I'm not sure that you really need to fire this outside of successful logins because I'm pretty sure that amplitude will do session tracking for us.

Reply:
so you're saying only log the auth event inside of Auth.login ?